Competition2017/src/competition/subsystems/BaseXCANTalonSpeedControlledSubsystem.java

```diff
@@ -139,6 +139,10 @@ public boolean isAtSpeed() {
         return Math.abs(getSpeed() - systemTargetSpeed.get()) <= systemSpeedThresh.get();
     }
     
+    public boolean isAtTracerSpeed(){
+        return Math.abs(getSpeed()/2 - systemTargetSpeed.get()/2) <= systemSpeedThresh.get()/2;
```

Comment:
I'm a little confused about this logic, if everything is halved is it any different?